file-server/src/main/java/edu/lysak/fileserver/server/FileStorage.java

```diff
@@ -1,26 +1,81 @@
 package edu.lysak.fileserver.server;
 
+import edu.lysak.fileserver.util.SerializationUtils;
+
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.security.SecureRandom;
+import java.util.HashMap;
+import java.util.Map;
+
+import static edu.lysak.fileserver.util.Constants.SERVER_STORAGE_FOLDER;
```

Comment:
A better way is to have a field with a base storage directory (folder) instead of using Hardcoded constant.